traffic_ops/traffic_ops_golang/cdn/capacity.go

```diff
@@ -166,21 +181,39 @@ func addCapacity(cap CapData, cacheStats CacheStats, crStates tc.CRStates, crCon
 		if !strings.HasPrefix(*cache.ServerType, string(tc.CacheTypeEdge)) {
 			continue
 		}
-		if len(stats.KBPS) < 1 || len(stats.MaxKBPS) < 1 {
-			continue
+		var kbps, maxKbps float64
+		for intName, intStats := range stats {
+			if len(intStats.KBPS) < 1 || len(intStats.MaxKBPS) < 1 {
+				continue
+			}
+			if checkServiceInterface(tx, intName, cacheName) {
```

Comment:
Can this *if* statement's be extracted to save a level of indentation?

Currently:
```golang
if condition {
    /* several lines of code */
}
```

Restructured:
```golang
if ! condition {
   continue
}
 /* several lines of code */
```